expertise/create_dataset.py

```diff
@@ -54,13 +54,32 @@ def get_paper_notes(self, author_id, dataset_params):
         notes_v2 = list(openreview.tools.iterget_notes(self.openreview_client_v2, content={'authorids': author_id}))
         return notes_v1 + notes_v2
 
+    def deduplicate_publications(self, publications):
+        deduplicated = []
+
+        # Build index of pub.original
+        original_ids = [getattr(pub, 'original', None) for pub in publications]
```

Comment:
what don't you do `pub.original` here?

Reply:
yeah, this code should be updated. please use `pub.original` from now on. No need to changes other part of the code you are editing now.